### PR TITLE
Adds a chain around sympy for symbolic math

### PR DESCRIPTION
**Pull request by sreiswig**
- Description: Adds a new chain that acts as a wrapper around Sympy to give LLMs the ability to do some symbolic math.
- Dependencies: SymPy

### CODE REVIEW

File: langchain/chains/llm_symbolic_math/base.py [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+    Example:
+        .. code-block:: python
+
+            from langchain import LLMSymbolicMathChain, OpenAI
+            llm_symbolic_math = LLMSymbolicMathChain.from_llm(OpenAI())
+    """
+
+    llm_chain: LLMChain
+    llm: Optional[BaseLanguageModel] = None
```
Comment (hwchase17) [inline langchain/chains/llm_symbolic_math/base.py:33]: if deprecated, lets not include

File: langchain/chains/llm_symbolic_math/base.py [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    output_key: str = "answer"  #: :meta private:
+
+    class Config:
+        """Configuration for this pydantic object."""
+
+        extra = Extra.forbid
+        arbitrary_types_allowed = True
+
+    @root_validator(pre=True)
+    def raise_deprecation(cls, values: Dict) -> Dict:
```
Comment (hwchase17) [inline langchain/chains/llm_symbolic_math/base.py:47]: dont include, same reason as above

**Review by hwchase17 — COMMENTED** [review]
nice this is really cool

seems cargo culted from LLMMath chain, but no need to keep around all the deprecated values

File: langchain/chains/llm_symbolic_math/base.py (hunk 1 above)
Comment (sreiswig) [reply langchain/chains/llm_symbolic_math/base.py]: Removed in latest commit

File: langchain/chains/llm_symbolic_math/base.py (hunk 2 above)
Comment (sreiswig) [reply langchain/chains/llm_symbolic_math/base.py]: Removed in latest commit

**Comment by baskaryan** [comment]
thanks @sreiswig!

**Pull request merged**